pcap.go

```diff
@@ -178,30 +271,56 @@ func (p *Pcap) Close() {
 	C.pcap_close(p.cptr)
 }
 
-func (p *Pcap) NextEx() (pkt *Packet, result int32) {
-	var pkthdr_ptr *C.struct_pcap_pkthdr
-	var pkthdr C.struct_pcap_pkthdr
+func (p *Pcap) NextEx(pktin *Packet) (pkt *Packet, result int32) {
+	if pktin == nil {
+		pkt = new(Packet)
+	} else {
+		pkt = pktin
+		pkt.Headers_cnt = 0
```

Comment:
`Headers_cnt`: use CamelCase for this. I'm also not sure (as hinted above) about why do we need this? What was wrong with putting this in a slice and taking `len()`?